portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,29 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson; 
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    ArrayList<String> messages = new ArrayList<String>();
+    messages.add("Harare");
+    messages.add("Bulawayo");
+    messages.add("Mutare");
+
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);
+
     response.setContentType("text/html;");
```

Comment:
I believe the content type needs to be changed to 'application/json' for it to work properly with script.js.